### DISCUSSION TITLE
Heat capacities from SGTE unary

### DISCUSSION THREAD

**Discussion by caglayanoaras** [pycalphad/pycalphad#315] [Q&A] [answered]
I want to get the heat capacities of elements from the sgte unary dataset ( [https://www.sgte.net/en/free-pure-substance-database](url) ). When I upload the tdb file, 
```python
import matplotlib.pyplot as plt
import numpy as np
from pycalphad import Database, calculate
db = Database('unary50.tdb')
```
I get a warning:

> /u/system/SLES12/soft/pyiron/dev/anaconda3/lib/python3.7/site-packages/pycalphad/io/tdb.py:965: UserWarning: The type definition character `I` was defined in the following phases: ['CBCC_A12', 'BCC_A2', 'FCC_A1', 'HCP_A3'], but no corresponding TYPE_DEFINITION line was found in the TDB.
  warnings.warn(f"The type definition character `{typechar}` was defined in the following phases: "

And when I try to get the heat capacity
```python
result = calculate(db, ['CA'], 'FCC_A1', T=(1, 1000, 0.1), P=101325, N=1, output='heat_capacity')
```
I get a condition error

> ConditionError: None of the passed phases (['FCC_A1']) are active. List of possible phases: ['LIQUID'].

I have no problem with LIQUID phases. I am quite new to tdb files and any help is appreciated.

Thanks

**Comment by richardotis** [comment, marked as answer]
You are missing the vacancy pseudo-component. Try `['CA', 'VA']` for the components argument.

**Comment by bocklund** [comment]
The missing VA component is common enough that the docs might need an FAQ item to link to or search for. It should have some/part of the error message about inactive phases so it's searchable. 

@caglayanoaras: One other thing to note is that pycalphad will not automatically extrapolate outside of the Piecewise temperature limits like you may be used to in other software. That is, the TDB defines 

```
FUNCTION GHSERCA  298.15
 -4955.062+72.794266*T-16.3138*T*LN(T)-11.10455E-3*T**2-133574*T**(-1);
 1115.00 Y
 -107304.428+799.982066*T-114.2922467*T*LN(T)+23.733814E-3*T**2-1.2438E-6*T**3
 +18245540*T**(-1);  3000.00 Y
 -3703.12+192.63995*T-35.0*T*LN(T); 3001 N !
```

meaning that below 298.15 K and above 3001 K, the Gibbs energy will be zero.
